### PR TITLE
V7 rename

### PR DESCRIPTION
**Pull request by jacob-ebey**

### CODE REVIEW

File: integration/helpers/cf-template/app/root.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import type { LinksFunction } from "@remix-run/cloudflare";
-import { cssBundleHref } from "@remix-run/css-bundle";
+import type { LinksFunction } from "@react-router/cloudflare";
+import { cssBundleHref } from "@react-router/css-bundle";
```
Comment (brophdawg11) [inline integration/helpers/cf-template/app/root.tsx:2]: I think the css-bundle package is obsolete in a `vite`-only world?

File: sgconfig.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ruleDirs:
+  - ./rules
+testConfigs:
+  - testDir: ./rule-tests
```
Comment (brophdawg11) [inline sgconfig.yml:4]: Is this supposed to be included?  I think this is leftover from the AST Grep run

File: integration/helpers/cf-template/app/root.tsx (hunk 1 above)
Comment (markdalgleish) [reply integration/helpers/cf-template/app/root.tsx:2]: We have a few loose ends like this lying around, so I think we can merge this as-is for now and do a broader cleanup later.

**Comment by markdalgleish** [comment]
The Windows test failure is unrelated to this PR, this just happens to be the first PR touching the `dev` package since adding cross-browser integration tests. We can ignore that test failure for now and solve it in a separate PR.

**Review by markdalgleish — APPROVED** [review]

**Review by pcattori — APPROVED** [review]

**Pull request merged**